tripy/tripy/frontend/module/layernorm.py

```diff
@@ -47,7 +54,10 @@ class LayerNorm(Module):
     eps: float
     """A value added to the denominator to prevent division by zero."""
 
-    def __init__(self, normalized_shape: int, dtype: datatype.dtype = datatype.float32, eps: float = 1e-5) -> None:
+    correction: int
+    """Difference between the sample size and the degrees of freedom."""
+
+    def __init__(self, normalized_shape: Union[int, Tuple[int]], dtype: datatype.dtype = datatype.float32, eps: float = 1e-5, correction: int = 0) -> None:
```

Comment:
[Torch layernorm](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html) doesn't expose correction in its API, is there a specific use case in mind for this?

Reply:
Good point. We should probably remove this then. Are there any cases you can think of where this value is not = 0?

Reply:
I'm not aware of any such case where it would be modified. I think we should add both eps and correction to the constant fields - if a user finds a use case for changing the correction we can always change it later.

Reply:
I added normalized_shape but not `eps`. I think it makes sense for `eps` to be changed on the fly. This follows torch.